website/www/site/content/en/roadmap/java-sdk.md

```diff
@@ -17,9 +17,9 @@ limitations under the License.
 
 # Java SDK Roadmap
 
-## Next Java LTS version support (Java 21)
+## Next Java LTS version support (Java 25)
 
```

Comment:
we can keep this as the previous support version with the github issue and the Beam version. So the page could help us track the history.

Reply:
ok.